### PR TITLE
bottom navbar

### PR DESCRIPTION
**Pull request by ishak52**
check this please :smiley: @ashatat @ahmadmshatat @emanKhaledh

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  animation: App-logo-spin infinite 20s linear;
+  height: 80px;
+}
+
+.App-header {
```
Comment (ashatat) [inline frontend/src/App.css]: I can't see where do you use this

File: frontend/src/components/App.js [hunk 2]
```diff
@@ -1,40 +1,46 @@
...
-          {this.state.test.map((t)=> <p>{t.a}</p>
-          )}
-        </header>
-      </div>
-    );
-  }
-}
-
-export default App;
+// import React, { Component } from 'react';
```
Comment (ashatat) [inline frontend/src/components/App.js]: please delete uncommented code

**Comment by ishak52** [comment]
@emanKhaledh about the components thing you can replace it when you push your components .. if you had read the comments that I left in the code it should explain it to you " it's just for testing the navbar" :smile:

**Comment by ashatat** [comment]
Please solve the conflict @ishak52 

and what this commit from the ReadME change pull request doing here?
`add the file structured in the readme  2c6689f  `

**Pull request closed without merging**